addons/hr_attendance/models/hr_attendance_overtime.py

```diff
@@ -24,6 +25,26 @@ def _default_employee(self):
         string='Extra Hours (Real)', default=0.0,
         help="Extra-hours including the threshold duration")
     adjustment = fields.Boolean(default=False)
+    valid = fields.Boolean(
```

Comment:
What if I named it `active` instead? Probably even more generic, but it would make modifying views and adding `.filtered` unnecessary as that gets handled automatically for models with that field.